### PR TITLE
VIM-1341 implement gx command

### PR DESCRIPTION
**Pull request by DwordPtr**
If anything is not right here please let me know so I can fix it.
I absolutely love this plugin.

### CODE REVIEW

**Comment by AlexPl292** [comment]
Hi there! Great, this is a long-awaited feature. I've added a couple of comments.

**Comment by DwordPtr** [comment]
I just saw this.
I'll address them once I get back to my computer!

**Comment by GhengS** [comment]
why 'gx' open in browser not work?

**Comment by AlexPl292** [comment]
Hi @GhengS, This feature has not yet been merged with the IdeaVim plugin.

**Comment by DwordPtr** [comment]
@AlexPl292 I can't find the comments.

Would you mind checking the submit review button? I'm guessing because I've made this mistake often with gh's ui.

**Comment by AlexPl292** [comment]
Oh, I'm really sorry, my eyes were ignoring the `Pending` status of the comments 🤦 . Do you see the comments now?

**Comment by DwordPtr** [comment]
So I see your comments and have onced over how to address them.

I'll put some time into this when I'm not under deadline pressure at my day job.

File: annotation-processors/build.gradle.kts [hunk 1]
```diff
@@ -23,7 +23,7 @@ repositories {
 dependencies {
   compileOnly("com.google.devtools.ksp:symbol-processing-api:1.9.21-1.0.15")
   implementation("org.jetbrains.kotlinx:kotlinx-serialization-json-jvm:$kotlinxSerializationVersion") {
-    // kotlin stdlib is provided by IJ, so there is no need to include it into the distribution
+    // kotlin stdlib is provided by IJ, so there is no need to include it into the
```
Comment (chylex) [inline annotation-processors/build.gradle.kts:26]: Accidental change?

**Comment by AlexPl292** [comment]
Hi, this PR has been open for some time. Do you have plans to update it?

**Comment by DwordPtr** [comment]
@AlexPl292 
Sorry, I had a lot of personal life stuff happen over the last six months.

I would like to do this however it's gonna be a long time to implement this the proper way as Im new to text editors.

I think there's a couple options here.
  * use the older gx command as a holdover just to get vim parity
  * wait but merge another pr if someone gets to it before me.

Btw if you have some suggested reading for how to get up to speed quickly I'd gladly read it.

**Comment by AlexPl292** [comment]
Hi, I see this PR is almost a year old. I'll close it, but feel free to reopen it in case you'll restart the work on it!

**Pull request closed without merging**